src/main/java/org/prgrms/kdt/VoucherManagementController.java

```diff
@@ -0,0 +1,39 @@
+package org.prgrms.kdt;
+
+import org.prgrms.kdt.io.IO;
+import org.prgrms.kdt.model.CommandType;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Configuration;
+
+import static org.prgrms.kdt.utils.Constant.COMMAND_ERROR_PROMPT;
+import static org.prgrms.kdt.utils.Constant.COMMAND_LIST_PROMPT;
+
+@Configuration
+public class VoucherManagementController implements CommandLineRunner {
+    private final IO io;
+
+    public VoucherManagementController(IO io) {
+        this.io = io;
+    }
+
+
+    @Override
+    public void run(String... args) throws Exception {
+        while (true) {
+            io.doOutput(COMMAND_LIST_PROMPT);
```

Comment:
```suggestion
            io.doOutputCommandPrompt();
```
이런식으로 Io객체에 책임을 넘겨보면 어떨까요?

Reply:
변경완료했습니다! 
추가로 궁금한 것이 있습니다. 제가 고친 코드로는 특정 상황이 되면 IO에 책임을 넘겨줘야 함으로, 메소드를 새로 만들어야 
하는 상황이 생길 것 같습니다. 새로 메소드를 생성하는 방법 말고 다른 방법이 있을까요?

Reply:
`io.doOutput(COMMAND_LIST_PROMPT);`

새로 메서드가 생기는거에 문제가 있을까요?

두 메서드를 살펴보면, 위는 명령어 텍스트가 바뀌면(COMMAND_LIST_PROMPT) 현재 컨트롤러가 변경되어야하는데요. 

`io.doOutputCommandPrompt();`이 메서드에 메시지가 변경되면 `vouchermanagementcontroller.java`는 변경이 없을 것 같네요 ㅎㅎ